src/api/spec/data/project_log_entry_user_name.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+require_relative '../../db/data/20180214132015_project_log_entry_user_name.rb'
+
+RSpec.feature 'ProjectLogEntryUserName', type: :model do
+  let!(:user) { create(:confirmed_user) }
+  let!(:other_user) { create(:confirmed_user) }
+  let(:inexistent_user) { User.last.id + 42 }
+
+  describe 'up' do
+    let!(:entry1) { create(:project_log_entry_comment_for_project, user_name: user.id) }
+    let!(:entry2) { create(:project_log_entry_comment_for_package, user_name: other_user.id) }
+    let!(:entry5) { create(:project_log_entry_comment_for_package, user_name: inexistent_user) }
+    before do
+      ProjectLogEntryUserName.new.send(:up)
```

Comment:
` up ` is not a private method. So you don't need ` send ` here.

Reply:
Calling `ActiveRecord::Migration` internally needs some parameter, I couldn't figure out which one quickly so I used send instead.